### ISSUE TITLE
deprecation message

### ISSUE THREAD

**Issue by MARTIMM** [Raku/tap-harness6#42]
runnings raku version 
```
Welcome to 𝐑𝐚𝐤𝐮𝐝𝐨™ v2021.06-78-gd668d9929.
Implementing the 𝐑𝐚𝐤𝐮™ programming language v6.d.
Built on MoarVM version 2021.06-16-g74e828f0a.
```
I get the following message when testing
```
Saw 1 occurrence of deprecated code.
================================================================================
Method wait (from TAP::Result) seen at:
  /home/Data/root-opt/Raku/rakudo/install/share/perl6/site/sources/6FEC28FA0762AC132736C81C67CCDB823E5E83FA (TAP), line 148
Please use exitcode and/or signal methods (status is to be removed in 2022.06) instead.
--------------------------------------------------------------------------------
```

**Comment by JJ** [comment]
This will have to be addressed. Thanks for the report.

**Comment by Leont** [comment]
Fixed in #43
